app/views/devise/registrations/new.html.erb

```diff
@@ -6,6 +6,11 @@
     <%= devise_error_messages! %>
 
     <div class="field">
+      <%= f.label :username %><br />
+      <%= f.username_field :name, autocomplete: "off" %>
```

Comment:
Again, progress but not there yet: 
Making it f.text_field got rid of the error message, but the field isn't saving anything.
When I create a new game with the new sign up form including the name field, Game.last.host_player.email shows the email address, but Game.last.host_player.name returns nil.  (Game.last.host_player.password also returns nil.)
         What will it take to make the form actually SAVE the name??